plugins/UM3NetworkPrinting/resources/qml/DiscoverUM3Action.qml

```diff
@@ -371,7 +371,7 @@ Cura.MachineAction
 
```

Comment:
We need to inform the Connect team that we're making changes to their code for this.

Reply:
I've added it to their cross-team-review board :-)

Reply:
The board is a nice idea but please actually tell me in person in the future @rburema ;)

Reply:
I did go into the Connect room and told/warned people about it though, I think I saw you walk back into the room as I left :-)

... at which point I maybe should have told you as well.